### PR TITLE
Changing edge glow color according to UI

### LINKED ISSUE #1476: Accent color of the session details
**Actual Behaviour**

The primary and primary dark color for these sessions are different for each one of them but the accent color remains the same as of app i.e., Red (See the bubble at the bottom).

screenshot_2017-05-05-20-26-16

**Expected Behaviour**

This is a question as it may vary from my approach. Is this okay? I think it should match that of page.

**Steps to reproduce it**

Change the color for this activity.

**Would you like to work on the issue?**

Yes.

Target Milestone to solve the issue

https://github.com/fossasia/open-event-android/milestone/24

### PR DESCRIPTION
**Pull request by heysadboy**
Fixes #1476

The only thing that was distinguishing the unique colour of session detail activity was the edge glow of nested scrollview. So changing that instead.

Now you see that glow is of same colour as of the activity instead of red.

I had issues with earlier PR https://github.com/fossasia/open-event-android/pull/1577, so sending this instead.

### CODE REVIEW

**Comment by heysadboy** [comment]
@iamareebjamal Please review.

**Comment by heysadboy** [comment]
@m-murad I made this due to commit issues  and to make the changes asked in this closed one https://github.com/fossasia/open-event-android/pull/1577 .

**Comment by m-murad** [comment]
@heysadboy 
The commit message should tell the users what the PR is for.
It's not clear from your message.

I'm sorry, but you need to change this.

**Comment by heysadboy** [comment]
@m-murad Changing the commit message.

File: android/app/src/main/java/org/fossasia/openevent/activities/SessionDetailActivity.java [hunk 1]
```diff
@@ -465,4 +468,37 @@ public void onOffsetChanged(AppBarLayout appBarLayout, int verticalOffset) {
             isHideToolbarView = !isHideToolbarView;
         }
     }
+
+    /*Check if the version is above Lollipop to change the edge glow color according to the
+    session detail activity color*/
+
+    private void changeGlowColor(int color)
```
Comment (iamareebjamal) [inline android/app/src/main/java/org/fossasia/openevent/activities/SessionDetailActivity.java]: Please make it a static method in Views and use Java bracket notation

File: android/app/src/main/java/org/fossasia/openevent/utils/Views.java [hunk 2]
```diff
@@ -75,4 +82,13 @@ public static int getDarkColor(int color) {
         hsv[2] *= 0.8f;
         return Color.HSVToColor(hsv);
     }
+
+    @TargetApi(Build.VERSION_CODES.LOLLIPOP)
```
Comment (iamareebjamal) [inline android/app/src/main/java/org/fossasia/openevent/utils/Views.java]: You can just use `LOLLIPOP`, it is imported statically above

File: android/app/src/main/java/org/fossasia/openevent/activities/SessionDetailActivity.java (hunk 1 above)
Comment (iamareebjamal) [inline android/app/src/main/java/org/fossasia/openevent/activities/SessionDetailActivity.java]: The static method may have ScrollView as parameter

Comment (iamareebjamal) [inline android/app/src/main/java/org/fossasia/openevent/activities/SessionDetailActivity.java]: Also, please remove the extra line between comment and function

**Comment by iamareebjamal** [comment]
Please rebase and squash the commits

**Comment by iamareebjamal** [comment]
@heysadboy Please read these chats to learn about how to squash and rebase, instead of merging 

Thanks
https://github.com/fossasia/open-event-android/pull/1580

**Comment by heysadboy** [comment]
@iamareebjamal Thanks for helping. I have squashed the commits.

**Comment by heysadboy** [comment]
@iamareebjamal Do I have to do it again? I see two commits now.

**Comment by m-murad** [comment]
@heysadboy 
Its fine

**Pull request merged**